doc/project/working_groups.rst

```diff
@@ -0,0 +1,117 @@
+TSC Working Groups
+******************
+
+Overview
+########
+
+The TSC, at its discretion, may establish working groups or subcommittees to
+serve as focused teams dedicated to specific technical areas, initiatives, or
+tasks.
+
+Membership
+##########
+
+Working Group Membership Eligibility
+++++++++++++++++++++++++++++++++++++
+
+- Each Working group (WG) shall determine its own membership eligibility, in
+  consultation with the TSC.
+- Each working group shall have a core team of members who are actively involved
+  in its activities and decision-making processes.
+- It is expected that WG core membership shall be open to all Zephyr project
+  Contributors; however, working groups may impose restrictions such as the
+  number of participants from a single company.
```

Comment:
maximal or minimal number ?

Reply:
maximal? minimal is always 1?

Reply:
Well, you might want to say that:
- you want to have at least one from each company
- maximum X from a single company

I agree that the word "restriction" clarifies towards the sense of "maximal", but being fully clear wouldn't hurt.